### PR TITLE
added intro slides to splash screen UI

### LINKED ISSUE #52: Splash screen UI
a splash screen with inkling details about app 
Eg: https://www.androidhive.info/2016/05/android-build-intro-slider-app/

At last there will be signup and login page

### PR DESCRIPTION
**Pull request by sanxy**
Resolved #52

### CODE REVIEW

**Comment by sanxy** [comment]
Screenshot_20200421-134741
Screenshot_20200421-134755
Screenshot_20200421-134804
Screenshot_20200421-134809
Screenshot_20200421-134816
Screenshot_20200421-134822

**Comment by tanmayrauth** [comment]
@sanxy We don't need the first screen anymore.
1. Just keep 3 screen
2. Make back and next button similar to example
3. Please make some aesthetic improvements. Since it is first screen all impression about the app is dependent on this 
4. Make use of material design

**Review by tanmayrauth — COMMENTED** [review]
@sanxy We don't need the first screen anymore.
1. Just keep 3 screen
2. Make back and next button similar to example
3. Please make some aesthetic improvements. Since it is first screen all impression about the app is dependent on this 
4. Make use of material design

**Review by ankitshaw — APPROVED** [review]

File: components/lib/main.dart [hunk 1]
```diff
@@ -27,7 +21,7 @@ class _MyAppState extends State<MyApp> {
     // Start splash screen
     return new SplashScreen(
         seconds: 14,
-        navigateAfterSeconds: new MyHomePage(),
+        navigateAfterSeconds: new MyIntroPage(),
         title: new Text('Welcome To Travel Safe',
           style: new TextStyle(
               fontWeight: FontWeight.bold,
```
Comment (ankitshaw) [inline components/lib/main.dart]: as @tanmayrauth need to be removed, we don't need this now

**Review by ankitshaw — CHANGES_REQUESTED** [review]

**Comment by sanxy** [comment]
Screenshot_20200421-231001
Screenshot_20200421-231009
Screenshot_20200421-231013

**Comment by tanmayrauth** [comment]
Please increase the padding of the text horizontally. Refer the example. Also keep it 1-2 liner not more than that otherwise no one will read it.
Sorry but the bottom button and structure is still on inline with the example. 

Also make use of gradient colour overlay texture for background like this example:
image

You can see the background colour changes from top to botttom

**Comment by tanmayrauth** [comment]
@sanxy I appreciate you efforts for rapid development on changes suggested. Kudos to you.

File: components/lib/screens/introPage.dart [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            fontSize: 30.0,
+            fontWeight: FontWeight.bold,
+            fontFamily: 'RobotoMono'),
+        description: "Travel-safe simplifies safety for women in the digital world by"
+            "making it easy to stay connected to the nearest police station.",
+        styleDescription:
+        TextStyle(color: Color(0xffFFFFFF),
+            fontSize: 20.0,
+            fontStyle: FontStyle.italic,
+            fontFamily: 'Raleway'),
```
Comment (ankitshaw) [inline components/lib/screens/introPage.dart]: Don't use italic style keep it minimalistic.

File: components/lib/screens/introPage.dart [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+            fontFamily: 'RobotoMono'),
+        description: "When travelling alone, it's best to raise your guards, stay alert and make"
+            "use of the technology that entertains you and lies comfortably in your purse - your"
+            "mobile phone, for your safety.",
+        styleDescription:
+        TextStyle(color: Color(0xffFFFFFF),
+            fontSize: 20.0,
+            fontStyle: FontStyle.italic,
+            fontFamily: 'Raleway'),
+        pathImage: "images/phone.png",
```
Comment (ankitshaw) [inline components/lib/screens/introPage.dart]: try with open san  for the header and roboto for content text.. if possible keep the content small. 
@tanmayrauth  we may have to decide the content

**Review by ankitshaw — CHANGES_REQUESTED** [review]

**Comment by sanxy** [comment]
Screenshot_20200423-200057
Screenshot_20200423-200103
Screenshot_20200423-200109

**Comment by tanmayrauth** [comment]
1. Please make first screen colour like this 
image

2. Choose some different colour for other 2 screens even.
3. choose some different technology image as logo.
4. For location logo give make with location image. similar to this 
image

5. Horizontal line above the button just like example is missing.

**Comment by tanmayrauth** [comment]
can you give screenshots?

**Comment by sanxy** [comment]
Screenshot_20200424-193648
Screenshot_20200424-193655
Screenshot_20200424-193703

**Comment by tanmayrauth** [comment]
1. technology still needs a change
2. horizontal line still missing as example
3. last screen should have 2 button pointing to Login and signup UI.
4. Make last screen a bit more greener

**Comment by tanmayrauth** [comment]
@sanxy did you got chance for getting the changes done?

**Comment by sanxy** [comment]
Not yet

**Comment by sanxy** [comment]
Screenshot_20200429-162606
Screenshot_20200429-162611
Screenshot_20200429-162618

**Comment by tanmayrauth** [comment]
All looks good. Just register button should be at left and since the register ui is there in place can you add event to open register UI.

**Comment by tanmayrauth** [comment]
> All looks good. Just register button should be at left and since the register ui is there in place can you add event to open register UI.

Once this is done we will get it merged

**Comment by sanxy** [comment]
I must point out there is an import in the Register page which is not used which should be deleted. I will make the necessary changes and push it.

**Comment by sanxy** [comment]
Screenshot_20200429-180457
Screenshot_20200429-180514

**Comment by tanmayrauth** [comment]
@sanxy I meant like how login button was at extreme right.
Similarly Register button should be at left

Taking example of skip button in technology screen

**Comment by sanxy** [comment]
Screenshot_20200429-184535

**Review by ankitshaw — APPROVED** [review]

**Comment by ankitshaw** [comment]
@sanxy  there is merge conflict. can you please pull he latest updates to the development branch and push your branch again.  Make sure nothing breaks.

**Pull request closed without merging**